awxkit/awxkit/api/pages/credentials.py

```diff
@@ -310,7 +310,7 @@ def create(
             credential_type=CredentialType,
             user=None,
             team=None,
-            organization=Organization,
+            organization=None,
             inputs=None,
             **kwargs):
         payload = self.create_payload(
```

Comment:
`create_payload` will take care to provide the default Organization in case any of user, team or organization is provided. This is more to avoid having organization and user or team defined at the same time in case someone calls `create(user=user)` or `create(team=team)`